src/asciidoc/docs/distribute/app-store.adoc

```diff
@@ -29,9 +29,9 @@ To create your App Store listing,
 
 include::{includespath}/steps-dev-portal.adoc[tag=go2-dev-portal-templates]
 
-. Click *New App Template*.
+. In your list of templates/listings, click *New App Template*.
```

Comment:
Suggest removing this intro phrase. The slash phrase ("template/listings") is confusing and they're already on the screen. Hopefully, we'll get this UI language cleaned up soon so we won't have things link "App Templates" in the UI.
```suggestion
. Click *New App Template*.
```

Reply:
Fixed.